generators/entity-client/templates/react/src/main/webapp/app/entities/entity.tsx.ejs

```diff
@@ -404,7 +425,7 @@ export const <%= entityReactName %> = (props: I<%= entityReactName %>Props) => {
       </div>
       <%_ if (databaseType !== 'cassandra') { _%>
         <%_ if (pagination === 'pagination' || pagination === 'pager') { _%>
-        <div className={ <%= entityInstance %>List && <%= entityInstance %>List.length > 0 ? '' : 'd-none' }>
+        { props.totalItems && props.totalItems !== 0 ? (<div className={ <%= entityInstance %>List && <%= entityInstance %>List.length > 0 ? '' : 'd-none' }>
```

Comment:
Similar to the other review, this can just be `props.totalItems`
```suggestion
        { props.totalItems ? (<div className={ <%= entityInstance %>List && <%= entityInstance %>List.length > 0 ? '' : 'd-none' }>
```

Reply:
@ruddell yes thanks 🙂